packages/cli/src/commands/discovery/init.ts

```diff
@@ -0,0 +1,30 @@
+import fs from 'node:fs';
+import degit from 'degit';
+import { Command } from '@oclif/core';
+import { confirm } from '@inquirer/prompts'
+
+export default class Init extends Command {
```

Comment:
todo: this command should receive a `path` arg like the ones on https://github.com/vtex/faststore/pull/2372 and it should put the output of degit on the given path. Its default value should be `.`.